homeassistant/components/lovelace/cast.py

```diff
@@ -126,7 +126,7 @@ async def async_play_media(
         return False
 
     if "/" in media_id:
-        url_path, view_path = media_id.split("/", 1)
+        url_path, _, view_path = media_id.partition("/")
```

Comment:
Double check that url_path is the one meant to be without slashes.  (Yes, the new code has no functional change and is equivalent to the prior code.)